run-spread

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+# check dependencies
+if ! snap list snapcraft &>/dev/null; then
+    echo "snapcraft is not installed"
+    exit 1
+fi
+
+# Clean the snaps created in previous runs
+touch test-build
+rm -rf built-snap
+mkdir built-snap
+rm -f snapd_1337.*.snap snapd_1337.*.snap.keep
```

Comment:
I see it's from `build-aux/snap/snapcraft.yaml`:

```
        # general build tags, note that version 1337 is used only in CI and
        # triggers testing specific build tags which produce binaries that are
        # insecure for use in production systems
```

I think a comment in `run-spread` explaining this would be helpful as well.

Reply:
It means elite in leetspeak :wink:. It was very common in 90s. https://en.wikipedia.org/wiki/Leet

Reply:
@olivercalder which error do you see when you try snapcraft with a clean snapd env?